ccp/apis/v1/ccp_server/backend/builds/build_logs.py

```diff
@@ -94,6 +99,12 @@ def response(namespace, appid, jobid, desired_tag, build):
         build=str(build_logs),
         scan=all_scan_logs
     )
+
+    try:
```

Comment:
Maybe you can edit https://github.com/CentOS/container-pipeline-service/blob/openshift/ccp/lib/clients/git/client.py#L23-L25 itself instead and use it?